### PR TITLE
 PDF: Changes in Table of Content

### PR DESCRIPTION
**Pull request by vuilleumierc**
Closes https://github.com/openoereb/pyramid_oereb/issues/1213
Goes with https://github.com/openoereb/pyramid_oereb_mfp/pull/81/

### CODE REVIEW

**Review by marionb — APPROVED** [review]
nice work thanks!

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 1]
```diff
@@ -511,6 +511,11 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
             self._multilingual_text(item, 'Title')
             self._multilingual_text(item, 'Content')
 
+        extract_dict['DisclaimerGrundbuch'] = extract_dict['ExclusionOfLiability'][0]
```
Comment (voisardf) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py:514]: If you do not mind, I would prefer having all in english. 😃  
I suggest DisclaimerLandRegister

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 2]
```diff
@@ -511,6 +511,11 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
...
+        extract_dict['DisclaimerKbS'] = extract_dict['ExclusionOfLiability'][1]
+        self._flatten_object(extract_dict, 'DisclaimerGrundbuch')
+        self._flatten_object(extract_dict, 'DisclaimerKbS')
```
Comment (voisardf) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py:517]: Same here: DisclaimerPollutedSites

File: tests/contrib/print_proxy/resources/test_extract.json [hunk 3]
```diff
@@ -341,6 +341,20 @@
     }
   ],
   "ExclusionOfLiability": [
+    {
+      "Content": [
+        {
+          "Text": "Der Kataster der belasteten Standorte (KbS) wurde anhand der vom Bundesamt f\u00fcr Umwelt BAFU festgelegten Kriterien erstellt und wird fortw\u00e4hrend aufgrund neuer Erkenntnisse (z.B. Untersuchungen) aktualisiert. Die im KbS eingetragenen Fl\u00e4chen k\u00f6nnen vom tats\u00e4chlichen Ausmass der Belastung abweichen. Erscheint ein Grundst\u00fcck nicht im KbS, besteht keine absolute Gew\u00e4hr, dass das Areal frei von jeglichen Abfall- oder Schadstoffbelastungen ist. Bahnbetrieblich, milit\u00e4risch und f\u00fcr die Luftfahrt genutzte Standorte liegen im Zust\u00e4ndigkeitsbereich des Bundes. F\u00fcr weitere Informationen wenden Sie sich an die kantonale Altlastenfachstelle, Amt f\u00fcr Umweltschutz und Energie (www.aue.bl.ch).",
```
Comment (voisardf) [inline tests/contrib/print_proxy/resources/test_extract.json:347]: is the utf-8 escaped form here wanted?

**Review by voisardf — APPROVED** [review]
looks good, thanks! I suggest some attribute name changes to stay completely in english if you do not mind.

**Comment by vuilleumierc** [comment]
@voisardf Thanks for your input, I did the suggested changes (also in MFP). You are right, it's more consistent to have everything in English in the code. As for the UTF8 escape, it came for a copy-paste from existing test data but I removed it now

**Comment by voisardf** [comment]
thanks! If you give me another quarter of an hour I'll check the pdf extract locally with the latest changes.

**Comment by voisardf** [comment]
works as expected. 👍

**Pull request merged**